lib/rules/require-capitalized-comments.js

```diff
@@ -52,11 +93,41 @@ var assert = require('assert');
 module.exports = function() {};
 
 module.exports.prototype = {
-    configure: function(requireCapitalizedComments) {
+
+    configure: function(options) {
+        var optionName = this.getOptionName();
+
+        var isTrue = options === true;
+        var isObject = typeof options === 'object';
+
         assert(
-            requireCapitalizedComments === true,
-            'requireCapitalizedComments option requires a value of true or should be removed'
+            isTrue ||
+            isObject,
+            optionName + ' option requires the value `true` ' +
+            'or an object with String[] `allExcept` property'
         );
+
+        // verify first item in `allExcept` property in object (if it's an object)
+        assert(
+            !isObject ||
+            Array.isArray(options.allExcept) &&
+            typeof options.allExcept[0] === 'string',
```

Comment:
I'm going to do `every` rather than `some`, because it should be asserting "not some" are "not equal to string", so it becomes a double negative. Sound ok?